ext/bert/c/decode.c

```diff
@@ -14,9 +15,12 @@
 #define ERL_BIN           109
 #define ERL_SMALL_BIGNUM  110
 #define ERL_LARGE_BIGNUM  111
+#define ERL_ENC_STRING    112
+#define ERL_UNICODE_STRING 113
```

Comment:
I chose these values because they are  [used for the offset in the callbacks struct](https://github.com/tenderlove/bert/blob/encoding/ext/bert/c/decode.c#L116).  Would adding a comment and removing the ERL prefix be enough?